src/ProjectFileManager.cpp

```diff
@@ -1529,12 +1524,10 @@ bool ProjectFileManager::Import(
 #endif
 
       ImportProgress importProgress(project);
-      bool success = Importer::Get().Import(project, fileName,
-                                            &importProgress,
-                                            &WaveTrackFactory::Get( project ),
-                                            newTracks,
-                                            newTags.get(),
-                                            errorMessage);
+      std::optional<LibFileFormats::AcidizerTags> acidTags;
+      bool success = Importer::Get().Import(
+         project, fileName, &importProgress, &WaveTrackFactory::Get(project),
+         newTracks, newTags.get(), acidTags, errorMessage);
```

Comment:
`newTags` could be a local object, that will simplify things. the acid tags could be then extracted like this:
```
if(newTags.HasTag(TAG_ACID))
   ...
```
I wonder if it's a desired behaviour at all when all other tags get overwritten? But it is not related to the scope of this PR

Reply:
Not sure I understand: 
```
if(newTags.HasTag(TAG_ACID))
   acidTag = StringToAcidizerTags(newTags.GetTag(TAG_ACID))
```
is what was there previously. But then, `newTags` gets committed.
The loop metadata tags are clip-dependent, though, so re-using those of clip A for the import of clip B is wrong.
The other tags are
```
TAG_TITLE
TAG_ARTIST
TAG_ALBUM
TAG_TRACK
TAG_YEAR
TAG_GENRE
TAG_COMMENTS
TAG_SOFTWARE
TAG_COPYRIGHT
```
and to me it is ok that they are project-wide if they are used when exporting?

Reply:
On export tags are copied from the project tags, so answerd is yes.
What I meant was that by making newTags a local structure we could avoid unnecessary (and there no control on the user side over that) tag overwrites, currently it's a bit complicated:
```
auto cleanup = finally([&]{
   if ( !committed )
      Tags::Set( project, oldTags );
});
auto newTags = oldTags->Duplicate();
Tags::Set( project, newTags );
```

Reply:
Isn't this to satisfy this comment:
```
   // do the actual import, creating whatever tracks are necessary with
   // the WaveTrackFactory and calling the progress callback every iteration
   // through the importing loop
   // The given Tags and AcidizerTags structures may also be modified.
   // In case of errors or exceptions, it is not necessary to leave outTracks,
   // tags or acidTags unmodified. If resulting outTracks is not empty, then
   // each member of it must be a nonempty vector.
   virtual void Import(
      ImportProgressListener& progressListener, WaveTrackFactory* trackFactory,
      TrackHolders& outTracks, Tags* tags,
      std::optional<LibFileFormats::AcidizerTags>& acidTags) = 0;
      ```
in ImportPlugin.h ?